docs/testing-be.md

```diff
@@ -1,27 +1,59 @@
 # Backend testing
 
-## Django and Python tests
+## Writing tests
 
-To run the the full suite of Python tests using Tox, cd to the project root and
-then run:
+We have multiple resources for writing new unit tests for Django, Wagtial, and Python code:
+
+- [CFPB Django and Wagtail unit testing documentation](https://github.com/cfpb/development/blob/master/guides/unittesting-django-wagtail.md)
+- [The Django testing documentation](https://docs.djangoproject.com/en/1.11/topics/testing/overview/)
+- [The Wagtail testing documentation](http://docs.wagtail.io/en/v1.13.4/advanced_topics/testing.html)
+- [Real Python's "Testing in Django"](https://realpython.com/testing-in-django-part-1-best-practices-and-examples/)
+
+## Running tests
+
+To run the the full suite of Python tests using Tox, 
+make sure you are in the cfgov-refresh root and then run:
 
 ```
 tox
 ```
 
+This will run linting tests and unit tests with migrations.
```

Comment:
This runs the linter by default?

Reply:
Yes, it's the same as running `tox -e lint -e unittest-py27-dj111-wag113-slow`.